app/locales/en.js

```diff
@@ -248,11 +248,63 @@ export default {
     whyHaveAccount: 'Signed-in volunteers can keep track of their work and will be credited in any resulting publications.',
     signIn: 'Sign in',
     register: 'Register',
-    alreadySignedIn: 'Signed in as %(name)s',
     orThirdParty: 'Or sign in with another service',
     withFacebook: 'Sign in with Facebook',
     withGoogle: 'Sign in with Google'
   },
+  newAccountsPage: {
+    signIn: 'Sign in',
+    register: 'Register',
+    successfullySignedIn: `You've successfully signed in!`,
+    successfullyRegistered: `You've successfully registered!`,
+    alreadySignedIn: 'Signed in as %(name)s',
+    alreadySignedInLinks: {
+      gotoHomepage: 'Go to the homepage',
+      gotoProjects: 'Find a new project to explore'
+    }
+  },
+  signInForm: {
+    signIn: 'Sign in',
+    signOut: 'Sign out',
+    userName: 'User name or email address',
```

Comment:
I think for consistency with other strings like `incorrectDetails`, but this would be a new change from how it has been. I'm not sure, but for consideration...
```suggestion
    userName: 'Username or email address',
```